test/test_torch.py

```diff
@@ -13415,6 +13415,26 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    def test_index_add(self, device):
+        # test coverage for issue with atomic add:
```

Comment:
I don't understand what "test coverage for issue with atomic add" means. How about something like "we don't implement dtype.is_signed in core, see issue"

Reply:
oh, I meant to comment with a link to the relevant ticket, because I was changing the name and want to make it clear that what we're hitting with this test is atomic add